core/signer/signer.go

```diff
@@ -44,13 +44,13 @@ type Signer struct {
 
 // New creates a new instance of the signer.
 func New(realm string,
-	mapID int64,
-	tmap trillian.TrillianMapClient,
-	logID int64,
-	sths appender.Remote,
-	mutator mutator.Mutator,
-	mutations mutator.Mutation,
-	factory transaction.Factory) *Signer {
+mapID int64,
```

Comment:
Running `make presubmit` would take care of all of that and shows other possible errors :)